openlibrary/catalog/marc/marc_binary.py

```diff
@@ -2,7 +2,7 @@
 from unicodedata import normalize
 
 from openlibrary.catalog.marc import mnemonics
-from openlibrary.catalog.marc.marc_base import MarcBase, MarcException, BadMARC
+from openlibrary.catalog.marc.marc_base import MarcBase, MarcException, BadMARC, re_isbn
```

Comment:
There are too many changes in unrelated areas in this PR, this is `re_isbn` is an open issue and I believe the fix is to use `self.re_isbn`, but I haven't had time to test it properly. Splitting the out roughly by folder / code area would help reviewing and separating easy changes from ones that have more risk. Some parts of /catalog are unused, but other parts are critical to import. /scripts tend to have completely different concerns.

Reply:
Neither this class nor its superclass define a `self.re_isbn`.